### PR TITLE
Hacktoberfest 2020 update

### PR DESCRIPTION
**Pull request by kimestoesta**
Opening a pull request on this branch now so we can comment on any changes! I think we are super close.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <meta charset="utf-8" />
     <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
```
Comment (kimestoesta) [inline index.html]: When I looked at this style sheet it didn't include all the weights so I loaded as a file instead

File: index.html [hunk 2]
```diff
@@ -50,7 +50,7 @@ <h2 class="subheader">CIRCLECI & HACKTOBERFEST</h2>
 
               </div>
               <div class="col padded">
-                  <div class="col" id="logoImage">
+                  <div class="animation" id="logoImage">
```
Comment (kimestoesta) [inline index.html:53]: made this bigger

File: index.html [hunk 3]
```diff
@@ -169,36 +169,36 @@ <h3 class="subheader">PRIZES:</h3>
...
-                    <h3 class="subheader">special sponsor</h3>
+                  <div class="col">
+                    <h3 class="">Special sponsors</h3>
                     <ul id="sponsorList">
-                        <li><img src="img/DigitalOcean.png" alt="DigitalOcean"></li>
-                        <li><img src="img/HashiCorp.png" alt="HashiCorp"></li>
-                        <li><img src="img/Spotify.png" alt="Spotify"></li>
+                        <li><img src="img/DigitalOcean.png" style="max-width: 165px;" alt="DigitalOcean"</li>
+                        <li><img src="img/HashiCorp.png" style="max-width: 130px;" alt="HashiCorp"></li>
+                        <li><img src="img/Spotify.png" alt="Spotify" style="max-width: 120px;"></li>
```
Comment (kimestoesta) [inline index.html:191]: Yeah couldn't figure out how to make all their logos even without getting really specific which each of them.

File: index.html [hunk 4]
```diff
@@ -134,15 +134,15 @@ <h3 class="subheader">PRIZES:</h3>
...
+                              <h3 class="">Prizes:</h3>
                               <ul>
-                                  <li>A Sticker pack and limited edition orb magic pin!</li>
+                                  <li>Sticker pack</li>
+                                  <li>T-shirt</li>
+                                  <li>Limited edition orb magic pin!</li>
                               </ul>
                             </div>
-                        </section>
-                        <section class="row padded col1-2 centered">
```
Comment (kimestoesta) [inline index.html]: Changed the layout since now there's only 2 Challenges.

File: index.html [hunk 5]
```diff
@@ -169,36 +169,36 @@ <h3 class="subheader">PRIZES:</h3>
...
+                        <li><img src="img/HashiCorp.png" style="max-width: 130px;" alt="HashiCorp"></li>
+                        <li><img src="img/Spotify.png" alt="Spotify" style="max-width: 120px;"></li>
                     </ul>
                   </div>
-                  <div class="col">
+                  <div class="col2">
                       <a href="#" class="btn btn-outline">Learn More</a>
                   </div>
                 </div>
+                <!--
```
Comment (kimestoesta) [inline index.html:198]: commented out the unconfirmed october events

**Review by kimestoesta — COMMENTED** [review]
Over all changes
- Tweaked text styles - used H3 vs subhead 
- small tweaks to sizes and spacing in style sheet
- changed where the navigation jumped

**Comment by jerdog** [comment]
@KyleTryon this is ready to get merged into master as the next to last step to go LIVE! 💯 🎉

**Pull request merged**